tools/version.sh

```
@@ -62,7 +62,7 @@ get_version_vars() {
 
     # Use git describe to find the version based on tags.
     #if [[ -n ${KUBE_GIT_VERSION-} ]] || KUBE_GIT_VERSION=$("${git[@]}" describe --tags --match='v*' --abbrev=14 "${KUBE_GIT_COMMIT}^{commit}" 2>/dev/null); then
-    if [[ -n ${KUBE_GIT_VERSION-} ]] || KUBE_GIT_VERSION=$(echo "v1.34.0+${KUBE_GIT_COMMIT}"); then
+    if [[ -n ${KUBE_GIT_VERSION-} ]] || KUBE_GIT_VERSION=$(echo "v1.35.1+${KUBE_GIT_COMMIT}"); then
```

Shellcheck (0.11.0)

[style] Useless echo? Instead of 'cmd $(echo foo)', just use 'cmd foo'.

(SC2116)